aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteStorageAdapter.java

```diff
@@ -783,6 +851,14 @@ private Completable updateModels() {
         return gson.getAdapter(itemClass).fromJson(modelInJsonFormat);
     }
 
+    private String getModelName(@NonNull Model model) {
```

Comment:
Ditto here, at least should be a utility class. But I'm also suspecting we don't need this all over the place, and can just keep it at the ingress/egress points, to the Flutter codebase.